web3/providers/eth_tester/middleware.py

```diff
@@ -182,7 +182,6 @@ def is_hexstr(value: Any) -> bool:
     "logs_bloom": "logsBloom",
     "state_root": "stateRoot",
     "receipts_root": "receiptsRoot",
-    "total_difficulty": "totalDifficulty",
```

Comment:
Ideally we'd want to remove this in eth-tester before removing it here.

Reply:
If no one is against it, I can open a PR in `eth-tester` to remove the `td` field.